index.bs

```diff
@@ -5830,6 +5830,74 @@ At this time, one [=credential property=] is defined: the [=resident key credent
+
+: Extension identifier
+:: `largeBlob`
+
+: Operation applicability
+:: [=authentication extension|Authentication=]
+
+: Client extension input
+::  <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        ArrayBuffer largeBlobWrite;
+        boolean largeBlobRead;
+    };
+    </xmp>
+
+    <div dfn-type="dict-member" dfn-for="AuthenticationExtensionsClientInputs">
+        :   <dfn>largeBlobWrite</dfn>
+        ::  An opaque byte string that the [=[RP]=] wishes to store with the existing credential.
+
+        :   <dfn>largeBlobRead</dfn>
+        ::  A boolean that indicates that the [=[RP]=] would like to fetch the previously-written blob associated with the asserted credential.
+    </div>
+
+: Client extension processing ([=authentication extension|authentication=])
+::
+       1. If {{AuthenticationExtensionsClientInputs/largeBlobRead}} is present and has the value `true`:
+           1. If the [[#sctn-getAssertion|assertion]] operation is successful, attempt to read any data associated with the asserted credential.
+           1. If successful, set {{AuthenticationExtensionsClientOutputs/largeBlob}} to the result.
+       1. If {{AuthenticationExtensionsClientInputs/largeBlobWrite}} is present, {{PublicKeyCredentialRequestOptions/allowCredentials}} contains exactly one element, and {{AuthenticationExtensionsClientInputs/largeBlobRead}} is not present, or has the value `false`:
+           1. If the [[#sctn-getAssertion|assertion]] operation is successful, attempt to store the contents of {{AuthenticationExtensionsClientInputs/largeBlobWrite}} on the [=authenticator=], associated with the indicated credential.
+           1. Set {{AuthenticationExtensionsClientOutputs/largeBlobWritten}} to `true` if successful and `false` otherwise.
+
+: Client extension output
+:: <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientOutputs {
+        ArrayBuffer largeBlob;
+        boolean largeBlobWritten;
+    };
+    </xmp>
+
+    <div dfn-type="dict-member" dfn-for="AuthenticationExtensionsClientOutputs">
+        :   <dfn>largeBlob</dfn>
+        ::  The opaque byte string that was associated with the credential identified by {{PublicKeyCredential/rawId}}. Only valid in the [=authentication extension=] case and only present if {{AuthenticationExtensionsClientInputs/largeBlobRead}} was `true`.
+
+        :   <dfn>largeBlobWritten</dfn>
+        ::  A boolean that indicates that the contents of {{AuthenticationExtensionsClientInputs/largeBlobWrite}} were successfully stored on the [=authenticator=], associated with the specified extension.
+    </div>
+
+
+: Authenticator extension input
+:: None.
+
+: Authenticator extension processing
+:: None.
+
+: Authenticator extension output
+:: None.
```

Comment:
But the blob would be stored in the authenticator, right? If so, I think it's worthwhile to point out that that is the case but the procedure is specified elsewhere (i.e., CTAP), like in #1424.

Reply:
Done in a way similar to #1424.